### PR TITLE
Module create function

### PR DESCRIPTION
**Pull request by samarthsharma36**
Indentations, docblock and locators added
@puneet0191 @yvesh

### CODE REVIEW

File: src/JoomlaBrowser.php [hunk 1]
```diff
@@ -1209,7 +1209,7 @@ public function setSiteOffline ($enable == true)
 	{
 		$this->debug('I set the site to offline');
 		$this->doAdministratorLogin();
-		$this->amOnPage('/administrator/index.php?option=com_config');
+		$this->amOnPage(Locators::$globalConfiguratinUrl);
 		if($enable == true){
```
Comment (yvesh) [inline src/JoomlaBrowser.php]: Code style

```php
if ($enable) 
{
```

Comment (samarthsharma36) [reply src/JoomlaBrowser.php]: I made the change @yvesh

**Comment by samarthsharma36** [comment]
Please review and let me know the changes @yvesh @puneet0191

**Comment by jatitoam** [comment]
@Samgithub1997 try updating your branch to the latest develop, it seems like you are repeating the code that was added in different commits and that's causing conflicts.  You will need to solve those manually

**Comment by Hackwar** [comment]
I've recreated this PR in #229, since this branch has been deleted and has conflicts. I'd be happy if you could test the other PR to see if everything still works.

**Pull request closed without merging**